### PR TITLE
Added routing to products. Initial products webpage developed. Added …

### PR DESCRIPTION
**Pull request by Cristopher-Artacho-WVSU**
-- Added routing to products page
-- Added ProductDiv components for reusability div in products
-- Initial Products webpage developed
-- Installed vite to configure and run app, affected some json files
-- Comments are provided in products page so that the reader can be guided

### CODE REVIEW

File: src/main.tsx
Comment (hydraadra112) [inline src/main.tsx:1]: What is happening here?

File: package-lock.json
Comment (hydraadra112) [inline package-lock.json:1]: I am assuming nag change ni because may gin run ka nga command? If so, what is it and why?

Comment (Cristopher-Artacho-WVSU) [reply package-lock.json:1]: installation and configurations to run npm dev run

File: src/main.tsx
Comment (Cristopher-Artacho-WVSU) [reply src/main.tsx:1]: Routing for products webpage

Comment (hydraadra112) [reply src/main.tsx:1]: LGTM. Closed.

File: package-lock.json
Comment (hydraadra112) [reply package-lock.json:1]: Ano nga mga commands cris?

Comment (Cristopher-Artacho-WVSU) [reply package-lock.json:1]: npm i kag npm i -D vite

Comment (hydraadra112) [reply package-lock.json:1]: LGTM. Closed.

**Pull request merged**